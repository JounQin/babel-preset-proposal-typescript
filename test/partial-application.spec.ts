import _ from './partial-application'
import { execute } from './helpers'

const proposal = 'partial-application'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_()).toEqual([3, 12])
})
