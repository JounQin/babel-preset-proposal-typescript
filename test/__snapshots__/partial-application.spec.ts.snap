// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`partial-application 1`] = `"addOne is not a function"`;
